instrumentation-core/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/TracerProperties.java

```diff
@@ -16,18 +16,24 @@
 
 package io.opentelemetry.instrumentation.spring.autoconfigure;
 
+import javax.validation.constraints.DecimalMax;
+import javax.validation.constraints.DecimalMin;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
 /**
  * Configuration for OpenTelemetry Tracer
  *
- * <p>Configures LoggingExporter and sets default tracer name
+ * <p>Sets default tracer name and sampler probability
```

Comment:
Let's go ahead and remove this, and instead add javadoc to at least the getters.

Reply:
Sorry. I forgot to clean up the java docs before creating this review. I'll fix this up now